src/test/java/library/repository/InMemoryBookRepositoryTest.java

```diff
@@ -0,0 +1,106 @@
+package library.repository;
+
+import library.domain.Book;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class InMemoryBookRepositoryTest {
+
+    private InMemoryBookRepository bookRepository;
+
+    @BeforeEach
+    void setUp() {
+        bookRepository = new InMemoryBookRepository();
+    }
+
+    @Test
+    @DisplayName("도서를 추가할 수 있어야 합니다.")
+    void testAdd() {
+        // Given
+        Book book = Book.createBook(1, "Book 1", "Author 1", 100, null, null);
+
+        // When
+        bookRepository.add(book);
+
+        // Then
+        List<Book> allBooks = bookRepository.findAll();
+        assertEquals(1, allBooks.size());
+        assertEquals(book, allBooks.get(0));
+    }
+
+    @Test
+    @DisplayName("도서 번호로 도서를 찾을 수 있어야 합니다.")
+    void testFindByBookNumber() {
+        // Given
+        Book book1 = Book.createBook(1, "Book 1", "Author 1", 100, null, null);
+        Book book2 = Book.createBook(2, "Book 2", "Author 2", 200, null, null);
+        bookRepository.add(book1);
+        bookRepository.add(book2);
+
+        // When
+        Optional<Book> foundBook = bookRepository.findByBookNumber(2);
+
+        // Then
+        assertTrue(foundBook.isPresent());
+        assertEquals(book2, foundBook.get());
+    }
+
+    @Test
+    @DisplayName("제목으로 도서를 찾을 수 있어야 합니다.")
+    void testFindListContainTitle() {
+        // Given
+        Book book1 = Book.createBook(1, "Sample Book 1", "Author 1", 100, null, null);
+        Book book2 = Book.createBook(2, "Book 2", "Author 2", 200, null, null);
+        Book book3 = Book.createBook(3, "Sample Book 3", "Author 3", 300, null, null);
+        bookRepository.add(book1);
+        bookRepository.add(book2);
+        bookRepository.add(book3);
+
+        // When
+        List<Book> foundBooks = bookRepository.findListContainTitle("Sample");
+
+        // Then
+        assertEquals(2, foundBooks.size());
+        assertTrue(foundBooks.contains(book1));
+        assertTrue(foundBooks.contains(book3));
```

Comment:
```suggestion
        assertThat(foundBooks.size()).isEqualTo(2);
        assertThat(foundBooks).contains(book1, book3);
```

assertJ의 문법에 contains를 제공하고 있습니다.

Reply:
[f81f6c2](https://github.com/prgrms-be-devcourse/java-library-management/pull/21/commits/f81f6c2ab9a11a34fe32a3839357ba27024fb270)
[f81f6c2](https://github.com/prgrms-be-devcourse/java-library-management/pull/21/commits/f81f6c2ab9a11a34fe32a3839357ba27024fb270)

위 경우를 포함하여 assertJ의 assertThat을 사용하도록 변경하였습니다